release.json

```
@@ -60,12 +60,12 @@
     },
     "release-a7": {
         "INTEGRATIONS_CORE_VERSION": "7.62.0-rc.1",
```

Waiting for the `7.62.0` for integrations-core here.
```suggestion
        "INTEGRATIONS_CORE_VERSION": "7.62.0",
```